### DISCUSSION TITLE
When using Prisma offline, how do I specify the engine cache directory?

### DISCUSSION THREAD

**Discussion by Lce-Lemon** [prisma/prisma#21418] [Q&A]
### Question

When using Prisma offline, how do I specify the engine cache directory?

Because I need to use Prisma offline, I have to pre-download the query engine and migration engine. Upon examining the source code, I noticed that when downloading the query engine and migration engine, they are cached in different directories for different executing users. Is it possible to make this cache directory a configurable option, allowing for the configuration of this cache directory so that multiple users can use the same cache?

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @Lce-Lemon 👋 

Thank you for raising this question.

This looks more like a feature request. Can you please open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) detailing your usecase so that our team can have a look.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
